### PR TITLE
[17.0][MIG] product_warranty: Migration to 17.0

### PR DESCRIPTION
**Pull request by peluko00**
Module migrated to version 17.0

cc https://github.com/APSL 155914
@miquelalzanillas @lbarry-apsl @javierobcn @mpascuall please review

- [Issue] https://github.com/OCA/rma/issues/376

### CODE REVIEW

**Review by mpascuall — APPROVED** [review]
Working good, checked in runboat

File: product_warranty/images/product_warranty.png
Comment (miquelalzanillas) [inline product_warranty/images/product_warranty.png:1]: Can you update this screenshot? 

Here is the updated one.

image

**Review by miquelalzanillas — APPROVED** [review]
LGTM

File: product_warranty/images/product_warranty.png
Comment (peluko00) [reply product_warranty/images/product_warranty.png:1]: Done, thanks!

**Comment by pedrobaeza** [comment]
The image was a bit outdated, hehe

/ocabot merge nobump

**Pull request merged**